bangazon-tinkr.ui/src/components/pages/Home/Home.js

```diff
@@ -12,12 +12,14 @@ class Home extends React.Component {
 
   searchTermChanged = (onChangeEvent) => {
     this.setState({ searchTerm: onChangeEvent.target.value });
-    if (this.state.selectedSearchType === 'product') {
+    if (this.state.selectedSearchType === 'products') {
       const filteredProducts = this.state.originalProducts.filter((product) => product.name.includes(onChangeEvent.target.value));
       this.setState({ filteredProducts });
-      this.selectedSearchType.router.push('/product/');
-    } else if (this.state.selectedSearchType === 'seller') {
-      // filter the users
+      this.selectedSearchType.router.push('/products/');
+    } else if (this.state.selectedSearchType === 'sellers') {
+      const filteredSellers = this.state.filteredSellers.filter((seller) => seller.sellerInfo.includes(onChangeEvent.target.value));
+      this.setState({ filteredSellers });
+      this.selectedSearchType.router.push('/sellers/');
     }
```

Comment:
Remove these lines and the break in the code should go away and meet all the functionality as described in the card